lib/apartment.rb

```diff
@@ -1,10 +1,9 @@
 require './lib/renter'
 
 class Apartment
+  attr_reader :number, :monthly_rent, :bathrooms, :bedrooms, :renter
 
-
-attr_reader :number, :monthly_rent, :bathrooms, :bedrooms, :renter
-  def initialize (unit)
+  def initialize(unit)
     @number = unit[:number]
```

Comment:
I have settings on my editor that will tell me when line breaks or indentations are incorrect.